test/ufuzz.js

```diff
@@ -135,23 +182,25 @@ var FUNC_TOSTRING = [
     "}();",
     ""
 ].join("\n");
+var loops = 0;
+var funcs = 0;
 
 function run_code(code) {
     var stdout = "";
     var original_write = process.stdout.write;
-    process.stdout.write = function(chunk) {
+    process.stdout.write = function (chunk) {
```

Comment:
Sorry, can you be more specific? Your comment is ambiguous to me. Did you mean;

- this particular line that has `process`
- just code according to the style guide already
- dont do this kind of whitespace-only update in the future
- d: something else?